src/styles/main.scss

```diff
@@ -24,14 +24,13 @@ li {
 
 .main {
   margin-top: 50px;
-  display: flex;
-  justify-content: center;
 }
 
 .container {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
-  flex-grow: 0;
-  max-width: 1200px;
+  margin: 0 auto;
+  max-width: ($product-card-size * $count-card) + (($count-card - 1) * $gap-card);
+  display: grid;
+  column-gap: 48px;
+  row-gap: 50px;
+  grid-template-columns: repeat(auto-fill, 200px);
```

Comment:
If you created the variables, than use them here and in the cap.
- And justify content in center.

Reply:
> And justify content in center.

 i have already content in center.